examples/TestDriver/src/pages/basics.js

```diff
@@ -14,63 +14,74 @@ import {
 import { Switch as NbSwitch } from 'native-base';
 import { connect } from 'react-redux';
 
-import Heap from '@heap/react-native-heap';
+import Heap, { HeapIgnore, HeapCaptureRestrictor } from '@heap/react-native-heap';
 import { incrementAction, decrementAction } from '../reduxElements';
 import { makeSentinelButton } from '../sentinelUtilities';
 
+const HeapIgnoredNbSwitch = Heap.withHeapIgnore(NbSwitch, {
+  ignoreInteraction: false,
```

Comment:
I think I know what you're saying here, but it's a funky name.  Perhaps `suppressAllTouchEvents` or something?  Is it limited to touch events?  I think so, right?

Reply:
This is just what was spec'd in the RFC.  I'm open to other names, but I'd want sign off from a few other people, too.

Reply:
That's fine.  I think my biggest complain was that it wasn't clear at first glance what the difference between ignoreInteraction and one of the things that came after it.

Reply:
I guess what's confusing is that I didn't realize you had a kind of an allowAll/disallowAll difference between the two - I just assumed that the HeapIgnore component would _of course_ ignore the interaction, so it doesn't even seem like it needs to be specified.  Possible I'm missing something, but if we go to the model where we only have a HeapIgnore component, does it make sense to keep this?

Reply:
By default, HeapIgnore will ignore everything.  If you want it to _not_ ignore some part of an interaction (e.g. don't ignore the interaction or target text), you would specify that you should not ignore those aspects (e.g. `ignoreInteraction={false} ignoreTargetText={false}`).

Would this be more intuitive if all these were named such that they'd be false by default (e.g. `captureInteraction={true} captureTargetText={true}` for the above example)?  This was an alternative that was mentioned in the [RFC](https://paper.dropbox.com/doc/RFC-React-Native-HeapIgnore-API--AcbarA5TfEJTVOJaJH0XnXe4Ag-FospQLIad9QNQGQjEC4cD#:uid=622472213691434445521435&h2=Alternative-Ideas) that natty seemed to think would be a good idea, but no one else gave their input on it.